redis/src/main/java/org/apache/calcite/adapter/redis/RedisDataProcess.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.adapter.redis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import redis.clients.jedis.Jedis;
+
+/**
+ * Data processing according to different data types.
+ */
+public class RedisDataProcess extends RedisDataBase {
```

Comment:
Why do we need both `RedisDataProcess` and `RedisDataBase` ? 

Is it possible to use a strategy interface instead of inheritance ? See [Prefer Composition Over Inheritance](https://stackoverflow.com/questions/49002/prefer-composition-over-inheritance)